src/test/ledger/PaymentSandbox_test.cpp

```diff
@@ -335,10 +341,14 @@ class PaymentSandbox_test : public beast::unit_test::suite
 public:
     void run ()
     {
-        testSelfFunding ();
-        testSubtractCredits ();
-        testTinyBalance ();
-        testReserve();
+        auto testAll = [this](auto&&... fs) {
+            testSelfFunding(fs...);
+            testSubtractCredits(fs...);
+            testTinyBalance(fs...);
+            testReserve(fs...);
+        };
+        testAll();
+        testAll(featureFlow, featureToStrandV2);
```

Comment:
This one doesn't need `test_bookdir(featureFlow)`?  If not it might be a good idea to include a comment why.